assets/js/pages/ClusterDetails/ClusterDetailsPage.jsx

```diff
@@ -81,47 +82,63 @@ export function ClusterDetailsPage() {
     case 'hana_scale_up':
     case 'hana_scale_out':
       return (
-        <HanaClusterDetails
+        <ClusterDetails
```

Comment:
This works for me we only have some duplicated props that we need to pass [clusterID, hosts, lastExecution, navigate] which is fine, i guess testing is easier that way. 
Just a note about
```
(clusterDetails={cluster.details})
```
In the HanaClusterDetails component we use
```
(details={cluster.details})
```

they are the same right? 

Maybe we can keep same prop name ? 
So we can  skip` <ClusterDetails clusterDetails={props.details} {...props}>` in the story
```
function ContainerWrapper({ children, ...props }) {
  return (
    <div className="max-w-7xl mx-auto px-4 sm:px-6 md:px-8">
      <ClusterDetails clusterDetails={props.details} {...props}>
        {children}
      </ClusterDetails>
    </div>
```

Reply:
okey

Reply:
done